features/steps/fakeapi.go

```diff
@@ -29,3 +31,7 @@ func NewFakeAPI() *FakeAPI {
 func (f *FakeAPI) Close() {
 	f.fakeHTTP.Close()
 }
+
+func (f *FakeAPI) setJSONResponseForGetPageData(statusCode int) {
+	f.fakeHTTP.NewHandler().Get("/data?uri=%2Feconomy%2Flatest&lang=en").Reply(statusCode).BodyString(`{"type": "article"}`)
+}
```

Comment:
You could make this more generic method so it could take either take the 'type' or a json string as the input. 

As for example, on one of the reference methods which is called 'getPageDataRequestToZebedeeFindsAWrongPageType', it's not actually getting a wrong page type it's finding a 404 which is slightly different.

Reply:
I see what you mean - I've changed it to get a wrong page type of "taxonomy_landing_page" now.